taskvine/src/bindings/python3/ndcctools/taskvine/dask_executor.py

```diff
@@ -87,6 +94,12 @@ def get(self, dsk, keys, *,
             low_memory_mode=False,
             checkpoint_fn=None,
             resources=None,
+            lib_resources=None,
+            lib_command=None,
+            import_modules=None,
+            dumb_env=None,
```

Comment:
Make this a permanent option with a good name, but just describe in a way that somebody won't use it by accident.

Reply:
Changed to env_per_task